README.md

```diff
@@ -13,16 +13,17 @@
 - In case of setting up synchronization, no whitespace is allowed in the path to the project directory.
 
 # Setup
-1. For Ubuntu,
+### For Ubuntu,
+1. Do
```

Comment:
I think we need to change `For Ubuntu` -> `Unix-like OS`. 

Among the bullet 1 - 4 here, only the bullet 1 is exclusively for the ubuntu and remaining 2 - 4 are valid to the macOS and other unix-like OS.